src/sage/geometry/hyperplane_arrangement/ordered_arrangement.py

```diff
@@ -0,0 +1,652 @@
+        if not self.is_linear():
+            raise TypeError('the arrangement is not projective')
+        if self._projective_fundamental_group:
+            return self._projective_fundamental_group
+        n = self.dimension()
+        r = len(self)
+        if n == 1:
+            return FreeGroup(0) / []
+        if n == 2:
+            G = FreeGroup(r - 1) / []
+            dic = {j: G.gen(j) for j in range(r - 1)}
+            dic[r - 1] = [prod(G.gens()) ** -1]
+            self._projective_fundamental_group = G
+            self._projective_meridians = dic
+            return G
+        if n == 3:
+            S = self.parent().ambient_space().symmetric_space()
+            coord = vector(S.gens())
+            Proj = ProjectivePlaneCurveArrangements(K,
+                                                    names=self.parent().variable_names())
+            L = Proj([vector(line.coefficients()[1:]) * coord for line in self])
+            G = L.fundamental_group()
+            self._projective_fundamental_group = G
+            self._projective_meridians = L._meridians_simpl
+            return G
+        H1 = self.hyperplane_section()
+        G = H1.projective_fundamental_group()
+        self._projective_fundamental_group = G
+        self._projective_meridians = H1._projective_meridians
+        return G
+
+    def projective_meridians(self):
+        r"""
+        Return the meridian of each hyperplane.
+
+        OUTPUT:
+
+        A dictionary
+
+        .. NOTE::
+
+           This functionality requires the ``sirocco`` package to be installed.
+
+        EXAMPLES::
+
+            sage: # needs sirocco
+            sage: A.<x, y> = OrderedHyperplaneArrangements(QQ)
+            sage: H = A(x, y, x + y)
+            sage: H.projective_meridians()
+            {0: x0, 1: x1, 2: [x1^-1*x0^-1]}
+
+            sage: # needs sirocco sage.graphs
+            sage: A3.<x, y, z> = OrderedHyperplaneArrangements(QQ)
+            sage: H = A3(hyperplane_arrangements.braid(4).essentialization())
+            sage: H.projective_meridians()
+            {0: [x2^-1*x0^-1*x4^-1*x3^-1*x1^-1],
+             1: [x3], 2: [x4], 3: [x1], 4: [x2], 5: [x0]}
+            sage: A4.<t1, t2, t3, t4> = OrderedHyperplaneArrangements(QQ)
+            sage: H = A4(hyperplane_arrangements.braid(4))
+            sage: H.projective_meridians()
+            {0: [x2^-1*x0^-1*x4^-1*x3^-1*x1^-1], 1: [x3],
+             2: [x4], 3: [x0], 4: [x2], 5: [x1]}
+
+            sage: # needs sirocco
+            sage: L.<t0, t1, t2, t3, t4> = OrderedHyperplaneArrangements(QQ)
+            sage: H = hyperplane_arrangements.coordinate(5)
+            sage: H = L(H)
+            sage: H.projective_meridians()
+            {0: [x2], 1: [x3], 2: [x0], 3: [x3^-1*x2^-1*x1^-1*x0^-1], 4: [x1]}
+        """
+        if not self._projective_meridians:
+            self.projective_fundamental_group()
+        return self._projective_meridians
```

Comment:
```suggestion
        if self._projective_meridians is None:
            self.projective_fundamental_group()
        return dict(self._projective_meridians)
```
Same comments as for affine meridians.